spartan/aztec-network/files/config/config-full-node-env.sh

```diff
@@ -7,6 +7,7 @@ if [ -n "$REGISTRY_CONTRACT_ADDRESS" ] && [ -n "$BOOTSTRAP_NODES" ]; then
 export BOOTSTRAP_NODES=$BOOTSTRAP_NODES
 export REGISTRY_CONTRACT_ADDRESS=$REGISTRY_CONTRACT_ADDRESS
 export SLASH_FACTORY_CONTRACT_ADDRESS=$SLASH_FACTORY_CONTRACT_ADDRESS
+export FEE_ASSET_HANDLER_CONTRACT_ADDRESS=$FEE_ASSET_HANDLER_CONTRACT_ADDRESS
```

Comment:
Dear lord almighty, save me from the env vars

![Bruce Almighty I Got The Power GIF by Jim Carrey](https://media0.giphy.com/media/fwta5c1e0O2hymy9p3/giphy.gif)